tools/publish-gcp-controller-manager.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This is ok in isolation, but I am worried about scalability of the build. I am really worried that shell script builds will creep up on us. How will this integrate with the full GCP release? e.g. https://github.com/kubernetes/cloud-provider-gcp/pull/71 . cc @ixdy

Reply:
Sticking with Bazel after offline discussion. While I personally prefer the go tool, most k8s devs are familiar with Bazel.